datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -65,6 +67,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SdxService.class);
 
+    @Value("${cb.enabledplatforms:}")
```

Comment:
this is the datalake module, use `datalake` prefix (or dl, I don`t know the proper prefix in the module)